src/service/plugins/battery.js

```diff
@@ -257,6 +257,64 @@ var Plugin = GObject.registerClass({
         });
     }
 
+    /**
+     * Notify the user the remote battery is at 60% charge.
+     */
+    _sixtyBatteryNotification() {
+        if (!this.settings.get_boolean('sixty-battery-notification'))
+            return;
+
+        // Offer the option to ring the device, if available
+        let buttons = [];
+
+        if (this.device.get_action_enabled('ring')) {
+            buttons = [{
+                label: _('Ring'),
+                action: 'ring',
+                parameter: null,
+            }];
+        }
+
+        this.device.showNotification({
+            id: 'battery|sixty',
```

Comment:
I think it would be a good idea to drop the `|level` ID suffix on these notifications, or change them all to `battery-level`. There should really only be one of these showing anyways and having multiple IDs complicates the logic of showing/hiding the proper one.

Reply:
I am trying to implement a spinbutton to replace both of these switches, which will drop this down to a single notification.